startMission

```diff
@@ -329,11 +333,13 @@ function addInventory()
 {
 	local objectName=$1
 	local objectLinkPath=$2
+	echo $objectName
+	echo $objectLinkPath
 	# echo $objectName >> $playerDir/.inventory
 	# ln $objectDir/$objectName $inventoryDir/$objectName
 	if [ ! -f "$inventoryDir/$objectName" ]
 	then
-		mv $objectLinkPath $inventoryDir/$objectName && echo $objectName >> $playerDir/.inventory
+		mv "$objectLinkPath" "$inventoryDir/$objectName" && echo $objectName >> "$playerDir/.inventory"
```

Comment:
This was working in all my other tests before you worked on revamping the object and inventory functionality, so let's make a note to test this thouroughly.